virttest/utils_netperf.py

```diff
@@ -99,26 +111,45 @@ def __init__(self, address, netperf_path, md5sum="", netperf_source="",
             self.netperf_path = os.path.join(self.netperf_dir,
                                              "src/netperf")
         else:
-            self.netserver_path = os.path.join(self.netperf_base_dir,
-                                               self.netperf_file)
-            self.netperf_path = os.path.join(self.netperf_base_dir,
-                                             self.netperf_file)
-
-        LOG.debug("Create remote session")
-        self.session = remote.remote_login(self.client, self.address,
-                                           self.port, self.username,
-                                           self.password, prompt,
-                                           linesep, timeout=360,
-                                           status_test_command=status_test_command)
+            self.netserver_path = self.__remote_file
+            self.netperf_path = self.__remote_file
```

Comment:
Hello @luckyh It should be the problem caused by this change.

Reply:
ok, could you please let me know about the error message you got? thanks!

Reply:
> ok, could you please let me know about the error message you got? thanks!

Hello @luckyh I have update it in bugzilla attachment.

Reply:
Have updated the patch, thanks!